examples/common/pigweed/protos/fabric_bridge_service.options

```diff
@@ -0,0 +1,6 @@
+chip.rpc.SynchronizedDevice.unique_id max_size:33
+chip.rpc.SynchronizedDevice.vendor_name max_size:33
+chip.rpc.SynchronizedDevice.product_name max_size:33
+chip.rpc.SynchronizedDevice.node_label max_size:33
+chip.rpc.SynchronizedDevice.hardware_version_string max_size:33
```

Comment:
hardware string can be up to 64 same with sw version string

![image](https://github.com/user-attachments/assets/3390ac86-72a7-4d4d-929d-4ca2c6f14fdc)

Reply:
missed those. Will correct this...z

Reply:
Fixed.